src/components/Pages/index.js

```diff
@@ -33,29 +44,40 @@ class VmDetailPage extends React.Component {
     let { match, vms, config } = this.props
     if (vms.getIn(['vms', match.params.id])) {
       return (<VmDetail vm={vms.getIn(['vms', match.params.id])} config={config} />)
-    } else {
-      if (vms.get('loadInProgress')) {
-        console.info('VmDetailPage: VM id can not be found: ', match.params.id, ' . Load is still in progress - wating before redirect')
-        return null
-      }
+    } else if (vms.get('loadInProgress')) {
+      console.info(`VmDetailPage: VM id can not be found: ${match.params.id}. Load is still in progress - waiting before redirect`)
+      return null
     }
 
-    console.info('VmDetailPage: VM id can not be found: ', match.params.id, ' . Redirecting to / ')
+    console.info(`VmDetailPage: VM id can not be found: ${match.params.id}. Redirecting to / `)
     return <Redirect to='/' />
   }
 }
-
 VmDetailPage.propTypes = {
   vms: PropTypes.object.isRequired,
   config: PropTypes.object.isRequired,
+
   getVms: PropTypes.func.isRequired,
```

Comment:
Move, please, functions to the bottom. We use convention that functions must be at bottom of `propTypes`.